pkg/reconciler/instances/istio/actions/mocks/testCommanderResolver.go

```diff
@@ -0,0 +1,16 @@
+package mock
+
+import "github.com/kyma-incubator/reconciler/pkg/reconciler/instances/istio/istioctl"
+
+type TestCommanderResolver struct {
```

Comment:
Why isn't this mock generated by `mockery`? We should stick to one convention.

Reply:
Because we then enter the circular dependency problem, unless we generate mocks in the same package

Reply:
Can you refactor it in the way to allow generating the mock?

Reply:
This one is hard. Let's leave it for a second round :)

Reply:
We decided to leave this one for future refactoring